src/java/org/apache/cassandra/cql3/statements/schema/AlterTypeStatement.java

```diff
@@ -235,14 +236,17 @@ public Raw(UTName name)
             this.name = name;
         }
 
-        public AlterTypeStatement prepare(ClientState state)
+        @Override
+        public AlterTypeStatement prepare(ClientState state, Function<String, String> keyspaceMapper)
         {
-            String keyspaceName = name.hasKeyspace() ? name.getKeyspace() : state.getKeyspace();
+            String keyspaceName = keyspaceMapper.apply(name.hasKeyspace() ? name.getKeyspace() : state.getKeyspace());
             String typeName = name.getStringTypeName();
 
             switch (kind)
             {
-                case     ADD_FIELD: return new AddField(keyspaceName, typeName, newFieldName, newFieldType);
+                case     ADD_FIELD:
+                    newFieldType.updateKeyspaceIfSet(keyspaceName);
```

Comment:
where does the `IfSet` suffix come from?
e.g. in `RawUT` we just set the keyspace and that's it

Reply:
yes, my bad, I'll fix it